jlm/llvm/opt/alias-analyses/Andersen.cpp

```diff
@@ -845,6 +852,27 @@ Andersen::Analyze(const RvsdgModule & module, util::StatisticsCollector & statis
   return result;
 }
 
+std::unique_ptr<PointsToGraph>
+Andersen::Analyze(const RvsdgModule & module, util::StatisticsCollector & statisticsCollector)
+{
+  auto pointsToGraph = AnalyzeModule(module, statisticsCollector);
+
+  // If a double check is requested, and the current configuration differs from the default naive
+  if (std::getenv(CHECK_AGAINST_NAIVE_SOLVER) != nullptr
+      && Config_ != Configuration::DefaultNaiveConfiguration())
+  {
+    // Temporarily switch to the default naive configuration
+    auto customConfig = Config_;
+    Config_ = Configuration::DefaultNaiveConfiguration();
+
+    auto naivePointsToGraph = AnalyzeModule(module, statisticsCollector);
+
+    Config_ = customConfig;
+  }
+
+  return pointsToGraph;
+}
+
```

Comment:
Did you forget the code for raising an error if the two points-to graphs diverge? Right now, the computed `naivePointsToGraph` is not used for anything (as far as I can see).